docs/UserGuide.md

```diff
@@ -35,100 +39,194 @@
 > * Items with `...` after them can have multiple instances.
 > * The order of parameters is fixed.
 
-#### Viewing help : `help`
-Format: `help`
 
-> Help is also shown if you enter an incorrect command e.g. `abcd`
- 
-#### Adding a person: `add`
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` 
+#### Changing storage location : `change`
+Changes the data file storage location. <br>
+Format: `change location to PATH`
+
+Examples:
+* `change location to C:\Users\Verbena\Dropbox\WhatNow`<br>
+Changes the data file storage location to C:\Users\Verbena\Dropbox\WhatNow<br>
+<img src="images/UI_Prototype/WhatNowUI-ChangeLocation-Colour.png" width="600">
+
+
+#### Adding a task: `add`
+Adds a task to WhatNow<br>
+Format: `add "DESCRIPTION" [on/by] [today/tomorrow/DAY/DATE] [from/at] [START_TIME] [to/till] [END_TIME] [every] [DAY/day/week/month/year] [low/medium/high]` 
+
+> * All task description should be written within double quotation marks.<br>
+Eg Add �buy eggs�. <br>
+> * The format for date must be *day month year* where year is optional.<br>
+Eg: 10th Oct, 4 November, 11th August 2017. Not 12/12/12. <br>
+> * Time should be in the *12 hour format*.<br>
+Eg: 10am, 6pm, 7.30pm. Not 7:30pm, 2359.
```

Comment:
We meant 7:30pm is not allowed but 7.30pm is allowed as in it should be a dot and not a colon.